<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>640</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>楷体</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_status">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>635</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>connection status: OK</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>90</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>端口：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>10</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>串口号</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>20</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>工作状态</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_state">
    <property name="geometry">
     <rect>
      <x>1122</x>
      <y>20</y>
      <width>81</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>10</y>
      <width>91</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QComboBox" name="PortBox"/>
     </item>
     <item>
      <widget class="QPushButton" name="OpenSerialButton">
       <property name="text">
        <string>打开串口</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_qgc_server_port">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="OpenButton">
       <property name="text">
        <string>打开端口</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>731</width>
      <height>28</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="lineEdit_roll"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_pitch"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_yaw"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_depth"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>731</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>横滚角</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>俯仰角</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>偏航角</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>当前深度</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget_video" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>651</width>
      <height>491</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>180</y>
      <width>451</width>
      <height>391</height>
     </rect>
    </property>
    <property name="title">
     <string>控制面板</string>
    </property>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>30</y>
       <width>82</width>
       <height>331</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>切换开关</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_19">
        <property name="text">
         <string>档位</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>下灯</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_20">
        <property name="text">
         <string>摇杆左右</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>旋钮左右</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>按钮#2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_21">
        <property name="text">
         <string>高度×1</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>20</y>
       <width>91</width>
       <height>351</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="lineEdit_switch_2"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_switch_key"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_knob_2"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_joy_2"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_knob_4"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_btn_2"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_rotate_cam_2"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>91</width>
       <height>351</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="lineEdit_switch_1"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_rotate_key"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_knob_1"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_joy_1"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_knob_3"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_btn_1"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_rotate_cam_1"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>92</width>
       <height>331</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>总开关</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>高度×100</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>上灯</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>摇杆前后</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>旋钮前后</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_18">
        <property name="text">
         <string>按钮#1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>调距</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="videoButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>130</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>重载视频</string>
    </property>
   </widget>
   <zorder>label_status</zorder>
   <zorder>label_7</zorder>
   <zorder>label_8</zorder>
   <zorder>label_9</zorder>
   <zorder>label_12</zorder>
   <zorder>label_13</zorder>
   <zorder>label_14</zorder>
   <zorder>label_15</zorder>
   <zorder>label_16</zorder>
   <zorder>label_17</zorder>
   <zorder>label_18</zorder>
   <zorder>label_23</zorder>
   <zorder>label_24</zorder>
   <zorder>lineEdit_state</zorder>
   <zorder>label_19</zorder>
   <zorder>label_20</zorder>
   <zorder>label_21</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>widget_video</zorder>
   <zorder>label_10</zorder>
   <zorder>label_11</zorder>
   <zorder>videoButton</zorder>
   <zorder>groupBox</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
